saas/views/metrics.py

```diff
@@ -110,27 +110,17 @@ class PlansMetricsView(ProviderMixin, TemplateView):
     (as a count of subscribers per plan per month)
     """
 
-    template_name = 'saas/plan_metrics.html'
+    template_name = 'saas/revenue_metrics.html'
```

Comment:
Yes, I modified my pull request.
For the plan metrics page, we use plan_metrics.html which is an extension of metrics_base.html.
For the revenue metrics page, we use metrics_base.html.